[2024-10-18T13:33:27.109+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-18T13:33:27.309+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.this_is_3rd_task scheduled__2023-12-03T00:00:00+00:00 [queued]>
[2024-10-18T13:33:27.358+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.this_is_3rd_task scheduled__2023-12-03T00:00:00+00:00 [queued]>
[2024-10-18T13:33:27.360+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-18T13:33:27.759+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): this_is_3rd_task> on 2023-12-03 00:00:00+00:00
[2024-10-18T13:33:27.833+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'this_is_3rd_task', 'scheduled__2023-12-03T00:00:00+00:00', '--job-id', '5091', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmppea14tre']
[2024-10-18T13:33:27.852+0000] {standard_task_runner.py:105} INFO - Job 5091: Subtask this_is_3rd_task
[2024-10-18T13:33:27.857+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=15679) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-18T13:33:27.873+0000] {standard_task_runner.py:72} INFO - Started process 15708 to run task
[2024-10-18T13:33:28.147+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v4.this_is_3rd_task scheduled__2023-12-03T00:00:00+00:00 [running]> on host ad4831ffc3d7
[2024-10-18T13:33:28.997+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sourabh' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='this_is_3rd_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-03T00:00:00+00:00'
[2024-10-18T13:33:29.083+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-18T13:33:29.436+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-18T13:33:29.449+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello world, this is my third task and will run after task1 and at same time task2']
[2024-10-18T13:33:29.677+0000] {subprocess.py:86} INFO - Output:
[2024-10-18T13:33:29.714+0000] {subprocess.py:93} INFO - Hello world, this is my third task and will run after task1 and at same time task2
[2024-10-18T13:33:29.717+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-18T13:33:30.448+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-18T13:33:30.454+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=this_is_3rd_task, run_id=scheduled__2023-12-03T00:00:00+00:00, execution_date=20231203T000000, start_date=20241018T133327, end_date=20241018T133330
[2024-10-18T13:33:30.697+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-18T13:33:31.010+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-18T13:33:31.033+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
